### PR TITLE
Add CFP content

### PR DESCRIPTION
**Pull request by daisylb**

### CODE REVIEW

File: data/Page/program.md [hunk 1]
```diff
@@ -3,138 +3,21 @@ title: Program
 order: 3
 ---
 
-# PyConline AU call for proposals
+## PyConline AU — Specialist Tracks
```
Comment (excitedleigh) [inline data/Page/program.md:6]:
```suggestion
# PyConline AU program

## Specialist tracks
```

File: data/Page/program.md [hunk 2]
```diff
@@ -3,138 +3,21 @@ title: Program
...
 
-We’d love you to present at the most online PyCon AU ever!
+Specialist Tracks are focused, themed “mini-conferences” that are curated by members of the community. This year we will have five fantastic tracks:
```
Comment (daisylb) [inline data/Page/program.md]:
```suggestion
Specialist tracks are focused, themed “mini-conferences” that are curated by members of the community. This year we will have five fantastic tracks:
```

Yes, I'm a pedant :P

File: data/Page/program.md (hunk 1 above)
Comment (daisylb) [inline data/Page/program.md]:
```suggestion
# Program

## Specialist tracks
```

File: data/Page/program.md [hunk 3]
```diff
@@ -3,138 +3,21 @@ title: Program
...
+* **DjancoCon Australia**: returning for their eighth year, this track is all things Django. Stories and fairy tales about Django, web development in Python, ponies, and unicorns 🦄!
+* **Security and Privacy**: How (and why) to be safe on the Big, Scary Internet, and avoid being the next headline data breach.
+* **Education**: Learn about learning, teach about teaching, and educate about education
+* **Data Science & Analytics**: Data and number crunching, analysis and visualisation, machine learning, and how those things affect us human beings.
+* **DevOops: Learning from Failure**: as their their inaugural PyConline appearance. “Learn from our failures, or teach us about yours”.
 
-PyConline AU's call for proposals (CFP) is open until **CLOSE DATE, anywhere on earth** (UTC-12; that is, midday UTC or 9:30pm Adelaide time on the following day). You can submit to the main conference and specialist tracks through the same process.
+PyConline AU's Call for Proposals will encompass submissions to all Specialist Tracks. You can submit your talk to the main conference and the Specialist Tracks with one click.
```
Comment (daisylb) [inline data/Page/program.md]:
```suggestion
PyConline AU's call for proposals will encompass submissions to all Specialist Tracks. You can submit your talk to the main conference and the specialist tracks with one click.
```

File: data/Page/program.md [hunk 4]
```diff
@@ -3,138 +3,21 @@ title: Program
...
 
-If you are looking for how to submit a talk, mentoring, or speaker benefits, you will find all of that information in our submission portal. A large button linking to the portal in a calming, brand-appropriate green colour has been placed below for your convenience.
+To learn more about the submitting to a Specialist Track, check out our [CFP](/speak).
```
Comment (daisylb) [inline data/Page/program.md]:
```suggestion
To learn more about the submitting to a specialist track, [check out our CFP](/speak).
```

Added more words to the link entirely for the purpose of embiggening the click target.

File: data/Page/program.md [hunk 5]
```diff
@@ -3,138 +3,21 @@ title: Program
...
 
-<div class='center-content'>
-    <a href="https://pretalx.com/pycon-au-2020/" class='btn btn-chonk'>
-        Submit to PyCon AU
-    </a>
-</div>
+## PyConline AU — Main Conference
```
Comment (daisylb) [inline data/Page/program.md]:
```suggestion
## Main conference
```

**Pull request closed without merging**